examples/js/exporters/GLTFExporter.js

```diff
@@ -48,6 +48,94 @@ var PATH_PROPERTIES = {
 	morphTargetInfluences: 'weights'
 };
 
+var DEFAULT_OPTIONS = {
+	mode: "glb",
```

Comment:
nit: Three.js code style allows both single quotation `'` and double quotation`"` for strings, but IMO the single one would be more preferable. The single one is used more many in Three.js and glTF exporter. (I think we should replace the existing double one with the single one for the consistency in the exporter someday.)